modules/elements-color-picker/assets/js/editor/commands/show-swatches.js

```diff
@@ -0,0 +1,157 @@
+import CommandBase from 'elementor-api/modules/command-base';
+import Utils from '../../../../../../core/app/assets/js/utils/utils';
```

Comment:
```suggestion
import Utils from 'elementor/core/app/assets/js/utils/utils';
```

Reply:
Nice! Is that a global path in webpack or something?